src/sentry/workflow_engine/endpoints/validators/error_detector.py

```diff
@@ -0,0 +1,46 @@
+from django.db import router, transaction
+from rest_framework import serializers
+
+from sentry import audit_log
+from sentry.api.fields.empty_integer import EmptyIntegerField
+from sentry.models.project import Project
+from sentry.utils.audit import create_audit_entry
+from sentry.workflow_engine.endpoints.validators import BaseGroupTypeDetectorValidator
+from sentry.workflow_engine.models.detector import Detector, ErrorDetector
+
+
+class ErrorDetectorValidator(BaseGroupTypeDetectorValidator):
+    groupingEnhancements = serializers.CharField(required=False, allow_blank=True, allow_null=True)
+    fingerprintingRules = serializers.CharField(required=False, allow_blank=True, allow_null=True)
+    resolveAge = EmptyIntegerField(
+        required=False,
+        allow_null=True,
+        help_text="Automatically resolve an issue if it hasn't been seen for this many hours. Set to `0` to disable auto-resolve.",
+    )
+
+    def create(self, validated_data):
+        with transaction.atomic(router.db_for_write(Detector)):
+            detector: ErrorDetector = ErrorDetector.objects.create(
+                project_id=self.context["project"].id,
+                organization_id=self.context["project"].organization_id,
+                name=validated_data["name"],
+                # no workflow_condition_group
+                type=validated_data["group_type"].slug,
```

Comment:
Since this is a proxy, and we'll need to re-lookup error detectors based on this field later,  i think this would need to be the enum value for error detector (Rather than the group_type slug)

(when processing the workflows, i lookup error detectors by type + project id, since the event won't be able to include the detector id)